WeakAurasOptions/RegionOptions/Group.lua

```diff
@@ -492,6 +492,23 @@ local function createOptions(id, data)
         WeakAuras.ResetMoverSizer();
       end
     },
+    scale = {
+      type = "range",
+      name = L["Group Scale"],
+      order = 45,
+      softMin = 0.05,
+      softMax = 2,
+      bigStep = 0.05,
+      get = function()
+        return data.scale or 1
+      end,
+      set = function(info, v)
+        data.scale = v
```

Comment:
You might want to move the group's anchor point here, so that the anchor point appears to not move.